packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -41,6 +42,7 @@ function MyFeedHeading({
   const shouldShowHighlightPulse = router.query?.hset === 'true';
   const { shouldUseFeedLayoutV1 } = useFeedLayout();
   const isV1Search = searchVersion === SearchExperiment.V1;
+  const onboardingOptimizations = useFeature(feature.onboardingOptimizations);
```

Comment:
I'm not sure though how to exclude anyone coming at this component not directly from onboarding...

Reply:
Maybe through query params? Or browser history? But still, yeah, this needs to be a component deeper if we are going to check those conditions.

Reply:
> this needs to be a component deeper if we are going to check those conditions.

Yup no matter what condition, means we might need to separate more on components if we want to do that...